### PR TITLE
Updated dependencies 

### PR DESCRIPTION
**Pull request by TendTo**
- updated dependencies (telegram 12.8 -> 13.4.1)
- removed unused import
- removed unused code
- use of lazy formatting for logging
- updated .gitignore

### CODE REVIEW

File: module/dev_functions.py [hunk 1]
```diff
@@ -1,77 +1,73 @@
...
+           "chat_type:"    + str(chat.type)+"\n" + "chat_title:" + str(chat.title) + "\n" + \
+           "___TESTO___\n" + \
+           "text:"     + str(text) + \
+           "date:"     + str(date) + \
+           "\n_____________\n"
+
+        log_tmp = open("logs/logs.txt", "a+")
+        log_tmp.write("\n" + message)
+    except Exception:
+        pass
```
Comment (Helias) [inline module/dev_functions.py:35]: I would suggest to print the exception

File: module/scraper_notices.py [hunk 2]
```diff
@@ -184,78 +178,81 @@ def get_notice_content(notice_dict, base_url, archive_p, notice_p):
...
-        logging.exception("Exception on call get_notice_content({}, {}, {}, {})".format(notice_dict, base_url, archive_p, notice_p))
+    except Exception:
+        logging.exception("Exception on call get_notice_content(%s, %s, %s, %s)", notice_dict, base_url, archive_p, notice_p)
         logging.exception(traceback.format_exc())
 
-def scrape_notices(context):
-    logging.info("Call scrape_notices({})".format(context))
 
-    logging.info("Starting scraping job")
+# -----------UNUSED-------------
```
Comment (Helias) [inline module/scraper_notices.py:186]: than.... may we could remove it

File: module/dev_functions.py (hunk 1 above)
Comment (TendTo) [reply module/dev_functions.py:35]: It is saved in the _logs/logs.txt_ file

Comment (Helias) [reply module/dev_functions.py:35]: no if it fails

Comment (TendTo) [reply module/dev_functions.py:35]: I mean, the whole method is very debug-oriented: it just logs all messages the bot receives in the file.
It doesn't do anything else, and is used only one time, in a MessageHandler

Comment (Helias) [reply module/dev_functions.py:35]: ok, I am too much pedantic

**Review by Helias — APPROVED** [review]

**Pull request merged**